src/main/java/de/tum/in/www1/artemis/web/rest/ParticipationResource.java

```diff
@@ -152,8 +150,7 @@ public ParticipationResource(ParticipationService participationService, Programm
 
         // if this is a team-based exercise, set the participant to the team that the user belongs to
         if (exercise.isTeamMode()) {
-            participant = teamRepository.findOneByExerciseIdAndUserId(exercise.getId(), user.getId())
-                    .orElseThrow(() -> new BadRequestAlertException("Team exercise cannot be started without assigned team.", "participation", "cannotStart"));
```

Comment:
Are we certain this alert is still displayed properly? I'm also not sure whether `Not Found` is better in this context than `Bad Request`. We require a team to be assigned, which does not exist then, but the problem does not seem to me like a "classic" missing entity, the exercise is found and a prerequisite not fulfilled. The opinion of other's would be useful here as well.